dissect/target/plugins/apps/other/env.py

```diff
@@ -29,8 +29,8 @@ def check_compatible(self) -> None:
         pass
 
     @export(record=EnvironmentFileRecord)
-    @arg("--env-path", help="path to scan environment files in", required=True)
-    @arg("--extension", help="extension of files to scan", default="env")
+    @arg("--env-path", type=str, help="path to scan environment files in")
+    @arg("--extension", type=str, help="extension of files to scan", default="env")
```

Comment:
```suggestion
    @arg("--env-path", type=str, required=True, help="path to scan environment files in")
    @arg("--extension", type=str, default="env", help="extension of files to scan")
```
Wouldn't this still need `required=True`?

Reply:
`--env-path` is `required=False` per the `--` logic, but it's a required argument for this plugin, so we need to explicitly set it `required=True`.